src/components/Layout/Layout.stories.tsx

```diff
@@ -56,3 +59,8 @@ export const RightSidebar = RightSidebarTemplate.bind({});
 RightSidebar.args = {
   variant: 'right-sidebar',
 };
+
+export const FiftyFifty = FiftyFiftyTemplate.bind({});
```

Comment:
super nit: it'd probably be good to write new stories in CSF v3 if possible

Reply:
Thought it would be weird to just write one story in CSF v3, so ended up turning all the stories in this file to CSF v3